presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -322,8 +325,17 @@ private static boolean checkSchemaMatch(org.apache.parquet.schema.Type parquetTy
             switch (prestoType) {
                 case ROW:
                     if (groupType.getFields().size() == type.getTypeParameters().size()) {
+                        checkArgument(type instanceof RowType, "It must be a RowType here.");
```

Comment:
do we need to check type here? prestoType is from type, which is ROW. I think we do not need this line

Reply:
good call, removed